sapi/fpm/fpm/fpm_sockets.c

```diff
@@ -386,6 +406,23 @@ static int fpm_socket_af_unix_listening_socket(struct fpm_worker_pool_s *wp) /*
 }
 /* }}} */
 
+#ifdef HAVE_SOCKETROUTE
+static int fpm_socket_setroute_init(void) /* {{{ */
```

Comment:
Is there a reason for not returning ``bool`` or ``zend_result``?

Reply:
no reason in particular indeed :-)